source/supplemental/installing/python_for_windows.rst

```diff
@@ -78,13 +78,13 @@ Add to the file (file should be empty):
 
 ::
 
-    PATH=$PATH:/C/Python37:/C/Python37/Scripts
+    PATH=$PATH:/C/Users/YourUserName/AppData/Local/Programs/Python/Python37:/C/Users/YourUserName/AppData/Local/Programs/Python/Scripts
 
-Note: your python executable may be located in a different path, to check the path you need to start windows shell (``cmd``) and type ``where python3`` - this command will output where python is currently installed.
+Note: your python executable may be located in a different path, to check the path you need to start windows shell (``cmd``) and type ``where python`` - this command will output where python is currently installed.
 
 Save the file and start a new gitbash shell.
 
-Once you have done that, you should be able to type ``python3`` at the command prompt, and get something like:
+Once you have done that, you should be able to type ``python`` at the command prompt, and get something like:
```

Comment:
So this is the trick: if someone has both python2 and python3 installed, it can get confusing. I used to encourage that folks set up their system to use "python" for python3, but:

https://www.python.org/dev/peps/pep-0394/

states the "python" should refer to python2 by default.

That's about *nix though. On Windows, I think folks are supposed to use the py.exe -- but then we have somethign different on Windows than *nix -- aarrgg!

I ended up putting "python3" in the OS-X instructions.

@hosungs: I'll defer to your judgement here -- so stick with "python" unless you say otherwise.

Reply:
Oh, then there is gitBash -- should that be "python3" -- double arrgg!

Reply:
On Windows, Python 3.x installation doesn't have any executable named "python3.exe", so entering "python3" on Windows after installing Python 3.x just fails.

And on gitbash, it doesn't have a separate binary, but just calls the python.exe, so again, entering "python3" on gitbash fails as well.

Besides, with gitbash on Windows, python.exe better be executed with "winpty", otherwise it might hang (it does on my case, so that's why I added the comment about creating an alias `alias python="winpty python"` in `.bashrc`. Hope all these clear up confusions with Windows Python installation...

Reply:
interesting on the windows laptop I tested this on python3 worked, it was a windows 10 OS